test/BoardView.cpp

```diff
@@ -135,8 +134,6 @@ TEST_CASE("BoardView UART", "[BoardView]") {
     auto uart0 = bv.uart_channels[0];
     REQUIRE(uart0.exists());
     REQUIRE(uart0.rx().exists());
-    REQUIRE(uart0.rx().exists());
-    REQUIRE(uart0.tx().exists());
```

Comment:
`rx` and `tx` are different

Reply:
ahh yes must have accidentally erased tx, will add that back